app/src/main/java/com/kickstarter/viewmodels/RewardViewModel.java

```diff
@@ -175,9 +175,9 @@ public ViewModel(final @NonNull Environment environment) {
       this.isClickable = isSelectable.distinctUntilChanged();
 
       this.startCheckoutActivity = this.projectAndReward
-        .compose(combineLatestPair(horizontalRewardsEnabled))
-        .filter(prAndHorizontalRewards -> !prAndHorizontalRewards.second)
-        .map(prAndHorizontalRewards -> prAndHorizontalRewards.first)
+        .compose(combineLatestPair(nativeCheckout))
```

Comment:
Actually, why would the normal `RewardsViewModel` know anything about native checkout? It's not shown if native checkout is enabled.